src/services/inplayer.subscription.service.ts

```diff
@@ -108,27 +111,29 @@ const formatCardDetails = (card: Card & { card_type: string; account_id: number
 
 // TODO: fix PurchaseDetails type in InPlayer SDK
 const formatTransaction = (transaction: InPlayerPurchaseDetails): Transaction => {
+  const purchasedAmount = transaction?.purchased_amount?.toString() || '0';
+
   return {
-    transactionId: transaction.parent_resource_id,
+    transactionId: transaction.parent_resource_id || 'access granted',
     transactionDate: transaction.created_at,
-    offerId: transaction.purchased_access_fee_id?.toString(),
+    offerId: transaction.purchased_access_fee_id?.toString() || '/',
```

Comment:
What is the slash for?

Reply:
Maybe add it to the translations. For English I would recommend 'Other', 'None', 'No Transaction', or something along those lines